src/App.js

```diff
@@ -77,6 +93,26 @@ const App = () => {
   };
   const onError = (error) => console.log("Error =", error);
 
+  const assuranceLevels = ["low", "medium", "high"];
+
+  const onLevelOfAssuranceChange = (event) => {
+    if (event.target.value) {
+      setLevelOfAssurance(
+        event.target.value !== levelOfAssurance ? event.target.value : ""
+      );
+    }
+  };
+
+  const metadataDevices = ["laptop", "mobile", "unknown"];
```

Comment:
Move this outside the component, please. I also wonder if the user should select this option or we should calculate it as this demo is like a real application the integrators can use as example.

Reply:
At the moment, it is the integrator who decides the metadata and therefore the antispoofing they want to use in the request for our services.
If we make it mandatory to read the METADATA and use it for SICAP secure version. We could enforce it in the future, wdyt?

Reply:
Metadata has been removed. I moved the LevelsOfAssurance out of the component tho.